spec/stored_queries/ontology/ontology_get_associated_samples.yaml

```diff
@@ -0,0 +1,50 @@
+# Get all samples reference this term
+
+name: ontology_get_associated_samples
+params:
+  type: object
+  required: [id, ts, "@onto_terms"]
+  properties:
+    id:
+      type: string
+      title: Document ID
+      description: Ontology id of the term you want to get all the ancestors of 
+    limit:
+        type: integer
+        default: 20
+        description: Maximum result limit 
+        maximum: 1000
+    offset:
+        type: integer
+        default: 0
+        description: Result offset for pagination
+        maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+    "@onto_terms":
+      type: string
+      title: Ontology terms collection name
```

Comment:
Can you explain this more and/or give some examples?

See example of a stored query with full descriptions:

https://github.com/kbase/relation_engine/blob/develop/spec/stored_queries/djornl/djornl_fetch_clusters.yaml